tests/conftest.py

```diff
@@ -121,8 +152,8 @@ def rnd_str(n):
     return ''.join(np.random.choice(list('abcdefghijklmnopqrstuvwxyz'), n))
 
 
-@pytest.fixture
-def config_test():
+@pytest.fixture(autouse=True)
+def test_config():
```

Comment:
I think the reason I named it config_test rather than test_config was so that it wouldn't be mistaken by the reader as an actual test that gets run.  (Not a big deal.)

Reply:
Makes sense. But it seems like the fixture tag took care of that.